libraries/transport/transportControl/transportControl.H

```diff
@@ -28,10 +28,17 @@ protected:
     void operator--() override;
 
 private:
+    enum class dtState
+    {   
+        NOT_SPECIFIED,
+        SHOULD_DECREASE,
+        SHOULD_MAINTAIN,
+        SHOULD_INCREASE
+
+    };
+
+    dtState dtState_;
     porousMixture& composition_;
-    scalarList maxDeltaY_;
-    scalarList maxDYdT_;
-    bool upToDate_;
 
     void update();
```

Comment:
Could we take the opportunity to now rename this function to `setDtState` (or something similar)?